docs/Machines.md

```diff
@@ -0,0 +1,108 @@
+# Manage Computational Resources
+
+By default, simulation requests are processed by a shared pool of machines serving multiple users. These machines are equipped with 4 cores each, and the pool is limited to a predefined capacity. If you require dedicated resources with custom configurations, the **Inductiva API** allows you to easily set up virtual machines reserved solely for your simulations. These are managed via machine groups, *i.e.*, groups of homogeneous machines with specific requested properties that can be started and terminated on demand. With the `MachineGroup` class, you can configure, start, and terminate machines. Creating a `MachineGroup` requires specifying the type of Virtual Machine to use and the number of such VMs that will compose the group. Currently, the available options for the `machine_type` are the ones available in the [Google Cloud Platform](https://cloud.google.com/compute/docs/machine-types). Once a `MachineGroup` is created, simply pass it as argument to your simulations, which will then be scheduled to run on those machines. 
+
+Note that a `MachineGroup` is literally a group of individual machines that do not communicate with each other. In other words, a `MachineGroup` is not a computational cluster where the load of each simulation is divided over all machines of the cluster.
+
+### Examples
+
+
+#### Running a simulation in a specific machine type
+
+```python
+
+import inductiva
+from inductiva import molecules
+
+# Create a MachineGroup object with a single machine of the desired type
+mg = inductiva.resources.MachineGroup(
+    machine_type="c2-standard-4",
+    num_machines=1,
+    disk_size_gb=40,
+)
+
+# Estimate the hourly cost (in $) of the cloud resources requested
+price_per_hour = mg.estimate_cloud_cost()
+
+# Start the machines
+mg.start()
+```
+
+Create your simulation scenario and run it on your machine group. Example with ProteinSolvation scenario:
+
+```python
+
+# Download the insulin protein (ID - "1ZNI") from RCSB database
+insulin_pdb_file = inductiva.molecules.utils.download_pdb_from_rcsb(pdb_id="1ZNI")
+
+scenario = molecules.ProteinSolvation(
```

Comment:
This does not exist anymore in this base package.
We will need to change this example asap.